src/router/RouteMeta.ts

```diff
@@ -31,6 +31,8 @@ export interface RouteMeta {
   title?: string
   // @TODO: REVIEW
   clickable?: boolean
+  index?: number
+  isLedger?: boolean
```

Comment:
whether the current active wallet is a *ledger wallet* should not be handled as part of the **routing** mechanism.

wallet types should be handled inside wallet scoped objects